x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -151,7 +152,7 @@ export function MlLatencyCorrelations({ onClose }: Props) {
               'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationColumnDescription',
               {
                 defaultMessage:
-                  'The impact of a field on the latency of the service, ranging from 0 to 1.',
+                  'The impact [0-1] of an attribute; the greater the impact, the more an attribute increases latency.',
```

Comment:
Not sure `impact` is appropriate here:

The ~~impact~~correlation score [0-1] of an attribute; the greater the ~~impact~~score, the more an attribute increases latency.

Reply:
Thanks!  I've made this change in 3f718962047723188f70cdf025e06da345c20769